staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go

```diff
@@ -783,7 +789,13 @@ func decode(codec runtime.Codec, versioner storage.Versioner, value []byte, objP
 
 // appendListItem decodes and appends the object (if it passes filter) to v, which must be a slice.
 func appendListItem(v reflect.Value, data []byte, rev uint64, pred storage.SelectionPredicate, codec runtime.Codec, versioner storage.Versioner) error {
-	obj, _, err := codec.Decode(data, nil, reflect.New(v.Type().Elem()).Interface().(runtime.Object))
+	var into runtime.Object
+	if u, ok := v.Addr().Interface().(runtime.Unstructured); ok {
```

Comment:
Same as above

Reply:
all real places that call this use `conversion.EnforcePtr(...)` which ensures addressability